tests/dicts/keypath/test_keypath_dict_list_wildcard.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from benedict.dicts import KeypathDict
+
+
+class keypath_dict_list_wildcard_test_case(unittest.TestCase):
+    def setUp(self):
+        self.blueprint = KeypathDict(
+            {
+                "a": [
+                    {"x": 1, "y": 1},
+                    {"x": 2, "y": 2},
+                ],
+                "x": [
+                    {"a": 10, "b": 10},
+                    {"a": 11, "b": 11},
+                ],
+            }
+        )
+
+    def test_correct_wildcard(self):
+        correct_wildcard_path_example = "a[*]"
+        self.assertEqual(self.blueprint[correct_wildcard_path_example], [1, 2])
```

Comment:
This is wrong, the expected result (all items of `a` item) are: `[{"x": 1, "y": 1}, {"x": 2, "y": 2}]`.

The result you expect should be obtained with this keypath: `a[*].x`.

Reply:
yes, forgot to rollback this change, I was interested what happens in this use case.